src/complex/Filter/Arguments.cpp

```diff
@@ -16,6 +16,11 @@ void Arguments::insert(std::string key, std::any&& value)
   m_Args.insert({std::move(key), value});
 }
 
+void Arguments::insert(const complex::StringLiteral& key, const std::any& value)
+{
+  m_Args.insert({std::move(key.str()), value});
```

Comment:
```suggestion
  m_Args.insert({key.str(), value});
```
The move here is unnecessary because `key.str()` is a r-value.